app/README.md

```diff
@@ -0,0 +1,2164 @@
+This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
```

Comment:
Remove this file. We usually keep one readme file in a project

Reply:
fixed